site/cypress/e2e/polls.spec.js

```diff
@@ -279,6 +279,68 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('.radio').eq(3).contains('Answer 2');
         cy.get('.radio > input').eq(3).should('not.be.checked');
 
+        // log into instructor, release histogram on poll closing, and close the poll
+        cy.logout();
+        cy.login();
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(6)').children().click();
+        cy.contains('Poll Cypress Test').siblings(':nth-child(1)').children().click();
+        cy.url().should('include', 'sample/polls/editPoll');
+        cy.get('#breadcrumbs > :nth-child(7) > span').should('have.text', 'Edit Poll');
+        cy.get('#student-histogram-release-setting').select('when_ended');
+        cy.get('#student-histogram-release-setting').invoke('val').should('eq', 'when_ended');
+        cy.get('#student-answer-release-setting').invoke('val').should('eq', 'never');
+        cy.get('button[type=submit]').click();
+
+        // log into student to verify histogram is released but answer is not
+        cy.logout();
+        cy.login('student');
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(3)').contains('View Poll').click();
+        cy.get('#toggle-info-button').should('be.visible');
+        cy.get('#toggle-histogram-button').should('be.visible').click();
+        cy.get('#poll-histogram').should('be.visible');
+        cy.should('not.contain', '#correctTag');
+
+        // log into instructor to able answer release on closed poll
+        cy.logout();
+        cy.login();
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(1)').children().click();
+        cy.get('#student-answer-release-setting').invoke('val').should('eq', 'never');
+        cy.get('#student-answer-release-setting').select('when_ended');
+        cy.get('#student-histogram-release-setting').invoke('val').should('eq', 'when_ended');
+        cy.get('#student-answer-release-setting').invoke('val').should('eq', 'when_ended');
+        cy.get('button[type=submit]').click();
+
+        // log into student and verify we can see both histogram and answer
+        cy.logout();
+        cy.login('student');
+        cy.visit(['sample', 'polls']);
+        cy.contains('Poll Cypress Test').siblings(':nth-child(3)').contains('View Poll').click();
+        cy.get('#toggle-info-button').should('be.visible');
+        cy.get('#toggle-histogram-button').should('be.visible').click();
+        cy.get('#poll-histogram').should('be.visible');
+        cy.get('#correctTag').should('be.visible');
```

Comment:
It would be good to verify both that the tag is visible, but also that it is next to the correct answer.  I think it would probably be fine to do this by searching for the text of the answer in the siblings of this element.